test/python/scripts/toolkits/delete_test_sc_files.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Script used/called by test_sc.py to delete the info.xml and toolkit.xml files generated for the tests
+# Also updates the info.xml and toolkit.xml files with random toolkit names, so different users can run tests concurrently.
+# Do NOT call this script from the cmd/terminal
+
+# Delete info.xml files for the toolkits located in /scripts/toolkits/
+cd toolkits
+for i in $(find . -name info.xml); do
+    rm $i
+done
+
```

Comment:
FYI - `xargs` is typically used with `find`.

`find toolkits -name info.xml | xargs rm`

Reply:
Resolved in #2232